### ISSUE TITLE
q2r parent calc copy error when using retrieved data 

### ISSUE THREAD

**Issue by mkotiuga** [aiidateam/aiida-quantumespresso#840]
When running the Q2rBaseWorkChain starting from retrieved data the DYN_MAT folder is copied within a folder called DYN_MAT

For example, running:

```ret_folder = load_node(1763568) # pk of DYN_MAT retrieved from a PhCalculation

param_q2r = Dict(dict={
    'INPUT': {"zasr":"simple"},
})

inputs = {'q2r': {
            'parent_folder' : ret_folder,
            'parameters':  param_q2r,
            'metadata': {
                'options': {
                    'resources': {'num_machines': 1,'num_cores_per_mpiproc':1},
                    'max_wallclock_seconds': 2*60
                },
                'description': 'phono_disp_test q2r'
            },
            'code': load_code(codename_q2r),
        }}
Q2RWC = WorkflowFactory('quantumespresso.q2r.base')
result = submit(Q2RWC, **inputs)
```

results in a error and a filestructure as follows

```
1779377  2m ago     Q2rBaseWorkChain        ⏹ Finished [300]
1779378  2m ago     Q2rCalculation          ⏹ Finished [330]

mkotiuga@theospc2:~/Desktop/aiida/aiida-quantumespresso$ verdi calcjob gotocomputer 1779378 
mkotiuga@theospc2:/scratch/mkotiuga/aiida/2d/34/1438-f338-4aef-8ddc-7f73aad6c0c2$ ls DYN_MAT/
DYN_MAT

```
This issue is coming from how the `FolderData` is copied [https://github.com/aiidateam/aiida-quantumespresso/blob/598a33fee8a9aa159553cf8b0881fec3cd4f87fa/src/aiida_quantumespresso/calculations/namelists.py#L180](url)

In essence `self._OUTPUT_SUBFOLDER ` is being copied into `self._OUTPUT_SUBFOLDER`

Changing:
` local_copy_list.append((source.uuid, self._OUTPUT_SUBFOLDER, self._OUTPUT_SUBFOLDER))`
to 
` local_copy_list.append((source.uuid, self._OUTPUT_SUBFOLDER, "."))`

fixes the issue, but I have not tested if this works generally.

**Comment by sphuber** [comment]
Thanks for the report @mkotiuga The line in question was added in https://github.com/aiidateam/aiida-quantumespresso/commit/22b27d2cace8210b88bf3489c27f263077ccae6d and released with v4.0.0. I will open a PR soon with the fix and make sure to add unit tests to verify it is the correct fix. Anyway we are about to release a patch version soon so we can fix the bug.

**Comment by sphuber** [comment]
@mkotiuga in your example script, what is `ret_folder`? Is that the `retrieved` output node of the `PhCalculation` or the `remote_folder`?

**Comment by sphuber** [comment]
Problem is actually due to a bug in `aiida-core` introduced in `v2.0`: https://github.com/aiidateam/aiida-core/issues/5647
I opened a PR to fix it. Once merged I will release v2.0.4 and then this problem here should be fixed.
